circle.yml

```diff
@@ -159,6 +159,21 @@ jobs:
           command: |
             codecov --required --gcov-exec "$GCOV" --gcov-root ~/build
 
+  gcc-32bit:
+    executor: linux
+    environment:
+      CMAKE_OPTIONS: -DTOOLCHAIN=cxx11-32bit
```

Comment:
It does not build because evmone requires c++17. Set the toolchain to cxx17-32bit (if such toolchain file does not exist create it accordingly)

Reply:
done.